src/explanation.rs

```diff
@@ -0,0 +1,184 @@
+use serde::{Deserialize, Serialize};
+
+use crate::{Prompt, Task};
+
+/// Input for a [crate::Client::explanation] request.
+pub struct TaskExplanation<'a> {
+    /// The prompt that typically was the input of a previous completion request
+    pub prompt: Prompt<'a>,
+    /// The target string that should be explained (i.e. influence of individual part
+    /// of the prompt should be indicated)
```

Comment:
This comment confuses me a bit.. Maybe indicate the usual case is the completion you got back from a model?
```suggestion
    /// The target string that should be explained (i.e. a completion from
    /// a previous completion request)
```